design/authn_azure/authn_azure_solution_design.md

```diff
@@ -412,6 +413,47 @@ Azure authenticator performance should conform with our other authenticators wit
 
 - DAP
 
+## Security
+
+### Security boundary
+
+#### Azure security boundary
+The security boundary (who can get an Azure access token) of the Azure Managed Identity Service is the resource to which it is attached to.
+For more details about this boundary, please visit [Azure managed identity security boundary](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/known-issues#what-is-the-security-boundary-of-managed-identities-for-azure-resources).
+
+#### Conjur security boundary
+The security boundary of Conjur Azure authenticator is the access token received in the authentication request.
+This access token is fetched using Azure Instance Metadata Service (IMDS) which in turn, uses Azure Active Directory (AAD) to generate the token.
+For the full details of how to achieve an access token and how its being generated, please visit [Azure managed identity overview](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview).
+
+
+### Authentication
+Conjur Azure authenticator authenticates the originator by its provided access token.
+The full authentication process described in the [Authentication flow](#authentication-flow) section.
```

Comment:
is described

Reply:
I am not so sure that it is the "Conjur Azure authenticator authenticates", in my head it is Conjur who authenticates and the application can use authn-azure as an authentication method